src/Apps/__tests__/Fixtures/Artist/Routes/ConsignRouteFixture.ts

```diff
@@ -0,0 +1,331 @@
+import { ConsignRoute_Test_QueryRawResponse } from "__generated__/ConsignRoute_Test_Query.graphql"
+import { ArtistConsignment } from "Apps/Artist/Routes/Consign/Utils/getConsignmentData"
+
+export const ConsignRouteFixture: ConsignRoute_Test_QueryRawResponse = {
```

Comment:
Where did this data come from? 

![data](https://user-images.githubusercontent.com/236943/77142250-be35cb80-6a3c-11ea-98ab-9aa1def48889.gif)

Reply:
Whoa, cool technique!!! 😍